src/Store.js

```diff
@@ -924,14 +924,16 @@ class Store {
           // Add all errors from the API response to the record(s).
           // This is done by comparing the pointer in the error to
           // the request.
-          json.errors.forEach((error) => {
-            const { index, key } = parseErrorPointer(error)
-            if (key != null) {
-              const errors = recordsArray[index].errors[key] || []
-              errors.push(error)
-              recordsArray[index].errors[key] = errors
-            }
-          })
+          if (Array.isArray(json.errors)) {
```

Comment:
Non blocking:
Is this something we need to add a ticket for on the backend? If it can be something other than an array, we should enforce it somewhere?

Reply:
can we perhaps coerce errors to be an array instead? or check for the existence of some other tell to make sure our errors are correct before trying to parse them?

Reply:
I think the errors may just be undefined in this scenario. I cannot recreate any other scenario.